usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -81,10 +81,14 @@ class AwardsElasticsearchDownload(_ElasticsearchDownload):
     _search_type = AwardSearch
 
     @classmethod
-    def query(cls, filters: dict) -> QuerySet:
+    def query(cls, filters: dict, values: List[str] = None) -> QuerySet:
         base_queryset = AwardSearchView.objects.all()
         flat_ids = cls._get_download_ids(filters)
-        queryset = base_queryset.extra(where=[f'"awards"."id" = ANY(SELECT UNNEST(ARRAY{flat_ids}::INTEGER[]))'])
+        queryset = base_queryset.extra(
+            where=[f'"vw_award_search"."award_id" = ANY(SELECT UNNEST(ARRAY{flat_ids}::INTEGER[]))']
```

Comment:
What promoted the change from `awards` to `vw_award_search`?

Reply:
For the Disaster Recipient endpoint I did not need to use any of the columns from the `awards` table and so it seemed pointless to have `vw_award_search` inner join to `awards` only for the purpose of this query when `vw_award_search` already has the `award_id` and has the `recipient_name` that I needed.